tools/gulptasks/dashboards/dist-upload.js

```diff
@@ -288,6 +292,22 @@ async function uploadZips(
  *
  * */
 
+/**
+ * Returns truncated versions
+ * @param {string} fullVersion the full semver string
+ *
+ * @return {[string, string]}
+ * Array containing truncated versions ordered from most to least specific, i.e. ["1.1", "1"]
+ */
+function getTruncatedVersions(fullVersion) {
+    const [major, minor] = fullVersion.split('.');
+
+    return [
+        `${major}.${minor}`,
+        major
+    ];
+}
+
```

Comment:
Can we merge this down with the other array? Seems to be a lot of code for one call and two array items.